content/en/js3/blocks/components/index.md

```diff
@@ -27,22 +26,11 @@ const film = {
 const createFilmCard = (film) => {
   const card = document.getElementById("film-card").content.cloneNode(true);
   // Now we are querying our cloned fragment, not the entire page.
-  card.querySelector('h3[name="title"]').textContent = film.title;
-
-  card.querySelector('p[name="director"]').textContent = `Director: ${film.director}`;
-
-  const times = document.querySelector('ul[name="times"]');
-  for (const time of film.times) {
```

Comment:
I think the "times" loop here is actually the most interesting thing in this example, mostly because it shows that even when using `template` sometimes it makes sense to mix in raw DOM APIs - what's your motivation in removing it?

Reply:
Oh one conversation in three places!!

For here, I paste for anyone interested

So, usually you do end up with a separate time creating function when building these cards. (I worked for 3 years building cinema and theatre listings.)  This is because handling time and making them accessible and internationalisable usually includes a bunch of other stuff as well. That's why I removed the list creation. It isn't how you would handle that issue. 

I could have converted it, like this, by putting a time template node in and then cloning and appending that
```
const timeTemplateNode = card.querySelector("time");
        times.map((time) => {
          const timeClone = timeTemplateNode.cloneNode();
          timeClone.textContent = time;
          timeTemplateNode.parentNode.appendChild(timeClone);
        });
        timeTemplateNode.remove();

```

or whatever. But that's not what you would really do with times, so I felt like it was too artificial and just bulking out the code samples  -- as yours was too and the bulk I felt was confounding and not clarifying.


Re the name attribute I removed: the attribute you would usually use to identify is data-
so data-director, data-times, data-certificate

is that part useful to wedge back in? Showing how to handle the same elements with different identifiers?

Reply:
Sounds good - thanks!